src/main/java/io/appium/java_client/service/local/AppiumServiceBuilder.java

```diff
@@ -296,7 +315,48 @@ private void loadPathToMainScript() {
         this.appiumJS = findMainScript();
     }
 
+    private String parseCapabilitiesIfWindows() {
+        String result = StringUtils.EMPTY;
+        final List<String> pathCapabilities = ImmutableList.of(AndroidMobileCapabilityType.KEYSTORE_PATH,
+                AndroidMobileCapabilityType.CHROMEDRIVER_EXECUTABLE, MobileCapabilityType.APP);
+
+        if (capabilities != null) {
+            Map<String, ?> capabilitiesMap = capabilities.asMap();
+            Set<? extends Map.Entry<String, ?>> entries = capabilitiesMap.entrySet();
+
+            for (Map.Entry<String, ?> entry : entries) {
+                Object value = entry.getValue();
+
+                if (value == null) {
+                    continue;
+                }
+
+                if (String.class.isAssignableFrom(value.getClass())) {
+                    if (pathCapabilities.contains(entry.getKey())) {
+                        value = "\\\"" + String.valueOf(value).replace("\\", "/") + "\\\"";
+                    } else {
+                        value = "\\\"" + value + "\\\"";
+                    }
+                } else {
+                    value = String.valueOf(value);
+                }
+
+                String key = "\\\"" + entry.getKey() + "\\\"";
+                if (StringUtils.isBlank(result)) {
+                    result = key + ": " + value;
```

Comment:
I usually prefer a ternary operator

Reply:
same above

Reply:
This was the code before commit #1337 so I decided to simply copy and paste this function to not break anyting.

Reply:
Should I still implement your suggestions?

Reply:
considering ternary operator, I disagree...usually makes code less readable.

Reply:
I don't have a strong opinion on that, just a matter of taste